### PR TITLE
Redesign Settings Page

### LINKED ISSUE #445: 2FA disable doesn't work
The 2FA disable confirm button doesn't actually do anything

### PR DESCRIPTION
**Pull request by freddyheppell**
## Overview of Changes
* Redesign views
  - [x] Settings
    - [x] Post-email change info
    - [x] Email change finished
  - [x] Security
    - [x] 2FA Setup Page
    - [x] 2FA disable page (also fixed #445)
    - [x] Backup code refresh confirmation page
    - [x] Backup code refresh new code page
  - [x] Donations
  - [x] Game Accounts
- [x] Move password change to security tab

## Deployment Requirements
- [ ] Composer update
- [ ] NPM update
- [ ] Database migration
- [ ] .env update
- [ ] Environment upgrade (eg. PHP version)
- [ ] Other (please specify)

### CODE REVIEW

File: resources/sass/v2/elements/_body.scss [hunk 1]
```diff
@@ -12,5 +12,7 @@ body {
     background-color: hsl(0, 0%, 94%);
     font-family: $font-body;
     font-size: 16px;
+    -webkit-font-smoothing: auto;
+    -moz-osx-font-smoothing: auto;
```
Comment (andyksaw) [inline resources/sass/v2/elements/_body.scss:16]: I wonder if that has any performance impact if it's applied to all text 🤔

File: resources/sass/v2/foundation/_dimensions.scss [hunk 2]
```diff
@@ -1 +1,5 @@
+$screen-m: 768px;
+$screen-l: 1024px;
+$screen-xl: 1300px;
```
Comment (andyksaw) [inline resources/sass/v2/foundation/_dimensions.scss:3]: 💯

File: resources/views/front/layouts/master.blade.php [hunk 3]
```diff
@@ -132,7 +132,7 @@ function gtag() {
                 <li>
                     <a href="#" class="nav-dropdown">Account <i class="fas fa-caret-down"></i></a>
                     <ul>
-                        <li><a href="{{ route('front.account.settings') }}">Account Settings</a></li>
+                        <li><a href="{{ route('front.account.profile') }}">Account Settings</a></li>
```
Comment (andyksaw) [inline resources/views/front/layouts/master.blade.php:135]: Shouldn't this go to the `v2` version?

File: resources/views/v2/front/components/navbar.blade.php [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <nav class="navbar">
     <div class="container">
         <div class="logo">
-            <img srcset="assets/images/logo-1x.png,
-                         assets/images/logo-2x.png 2x"
-                 src="assets/images/logo-1x.png"
+            <img srcset="/assets/images/logo-1x.png,
+                         /assets/images/logo-2x.png 2x"
+                 src="/assets/images/logo-1x.png"
```
Comment (andyksaw) [inline resources/views/v2/front/components/navbar.blade.php:6]: Hmm... I probably should've used `asset(...)`

**Review by andyksaw — APPROVED** [review]
There was a lot of code, so I only did a light look through, but overall LGTM!

File: resources/sass/v2/elements/_body.scss (hunk 1 above)
Comment (freddyheppell) [reply resources/sass/v2/elements/_body.scss:16]: As far as I know it's very minimal

**Pull request merged**